lib/fog/aliyun/compute.rb

```diff
@@ -316,7 +321,7 @@ def request(params)
                   end
           end
 
-          response.body = Fog::JSON.decode(response.body) if !response.body.empty? && response.get_header('Content-Type') == 'application/json'
```

Comment:
This value often comes back as `application/json; charset=utf-8`which fails the exact-string matching used before.